admin/server/projects.go

```diff
@@ -1203,15 +1205,16 @@ func (s *Server) SetProjectMemberUserRole(ctx context.Context, req *adminv1.SetP
 }
 
 func (s *Server) GetCloneCredentials(ctx context.Context, req *adminv1.GetCloneCredentialsRequest) (*adminv1.GetCloneCredentialsResponse, error) {
-	claims := auth.GetClaims(ctx)
-	if !claims.Superuser(ctx) {
-		return nil, status.Error(codes.PermissionDenied, "superuser permission required to get clone credentials")
-	}
 	observability.AddRequestAttributes(ctx,
 		attribute.String("args.org", req.Organization),
 		attribute.String("args.project", req.Project),
 	)
 
+	claims := auth.GetClaims(ctx)
+	if !claims.Superuser(ctx) {
```

Comment:
here force access flag is not added because its just a cli only API?

Reply:
It's because it's a sudo-only API (no normal users can call it). The `force_superuser_access` flag only makes sense for APIs that allow calls from normal users as well as superusers.